about.html

```diff
@@ -0,0 +1,240 @@
+            </div>
+        </div>
+    </section>
+
+    <section class="summit_info_wrapper">
+        <div class="summit_info">
+            <div class="summit_title_wrapper">
+                <div class="summit_title_container">
+                    <h2 class="summit_title">Creative Coders <br class="hide--desktop">
+                        Global Summit 2015 Logo</h2>
+                    <hr class="summit_bar">
+                </div>
+            </div>
+            <div class="summit_description_wrapper">
+                <div class="summit_description">
+                    <p class="hide--desktop">The logo of CC Global Summit 2015 was decided through the logo competition
+                        from 8, June to 7,
+                        July.
+                    </p>
+                    <p class="hide--mobile">
+                        The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7,
+                        July.
+                        After online public voting and Selection Committee’s judging process, the logo from Naresh
+                        Agrawal won the prize.
+                        After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit
+                        as follows:
+                    </p>
+                    <div class="summit_logo">
+                        <img class="logo_summit_desktop" src="./assets/images/logo.png" alt="about-logo">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="past_summit_wrapper">
+        <div class="past_summit">
+            <div class="past_summit_title_wrapper">
+                <div class="past_summit_title_container">
+                    <h2 class="past_summit_title">See the past CC Global Summits</h2>
+                    <hr class="past_summit_bar">
+                </div>
+            </div>
+            <div class="past_summit_description_wrapper">
+                <div class="past_summit_description">
+                    <p>
+                        Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw
+                    </p>
+                </div>
+            </div>
+
+            <div class="past_summit_img_wrapper_container">
+                <div class="past_summit_inner">
+                    <div class="past_summit_img_wrapper">
+                        <div class="past_summit_img_container">
+                            <div class="past_summit_img">
+                                <img src="./assets/images/past_summit.jpg" alt="" srcset="">
+                            </div>
+                            <div class="past_summit_info_wrapper">
+                                <div class="past_summit_info">
+                                    <p class="past_summit_date">2013</p>
```

Comment:
- [ ] Kindly use different years to show the past activities, I kindly recommend using the two ✌🏽 past years 2021 and 2022 to personalize your page.